src/client/components/ui/Link/Link.tsx

```diff
@@ -16,16 +18,24 @@ export function Link(props: LinkProps) {
 		children,
 		href,
 		prefetch = false,
+		title,
 		...rest
 	} = props;
 	const isInternalLink = useIsInternalLink(href);
 
+	function getTitle() {
+		if (isInternalLink) {
+			return title;
+		}
+		return title || `${getTextFromChildren(children)} - nouvelle fenêtre`;
```

Comment:
```suggestion
		return title ?? `${getTextFromChildren(children)} - nouvelle fenêtre`;
```
Si le dev met explicitement title à "" alors je pense qu'on ne veut pas l'ignorer